### DISCUSSION TITLE
How to get child info from a router

### DISCUSSION THREAD

**Discussion by ksathara** [openthread/openthread#8823] [Q&A] [answered]
Hi, 

I have a leader, router/routers and SEDs (Sleepy End Devices) in my network. I want to know 

1. How to retrieve connected child table and child info from a router (child's RSSI and link quality)
2. How to set a SED to act as a REED
3. Detect when a SED, which was connected to the leader, now connects to one of the routers 

Thanks heaps,
Kaushalya

**Comment by jwhui** [comment]
Using CLI on the parent, you can use the following commands:
- [`child table`](https://github.com/openthread/openthread/tree/main/src/cli#child-table)
- [`neighbor table`](https://github.com/openthread/openthread/tree/main/src/cli#neighbor-table)

**Comment by ksathara** [comment]
Thanks Jonathan,

When I try 'neighbor table' I only get children and routers connected to my
leader. I want to know the details of children connected to routers. Is
that possible?

I guess we can send those details from the router to the leader under app
control.

Cheers,
Kaushalya

**Reply by jwhui** [reply, marked as answer]
Retrieving topology information from other nodes requires querying over the network. Take a look at:
- https://github.com/openthread/openthread/pull/8682
